core/trino-main/src/main/java/io/trino/operator/TableScanWorkProcessorOperator.java

```diff
@@ -264,20 +267,16 @@ void close()
             implements WorkProcessor.Process<Page>
     {
         final ConnectorPageSource pageSource;
-        final LocalMemoryContext memoryContext;
 
-        ConnectorPageSourceToPages(AggregatedMemoryContext aggregatedMemoryContext, ConnectorPageSource pageSource)
+        ConnectorPageSourceToPages(ConnectorPageSource pageSource)
         {
             this.pageSource = pageSource;
-            this.memoryContext = aggregatedMemoryContext
-                    .newLocalMemoryContext(TableScanWorkProcessorOperator.class.getSimpleName());
         }
 
         @Override
         public ProcessState<Page> process()
         {
             if (pageSource.isFinished()) {
-                memoryContext.close();
```

Comment:
who will now close the context?

Reply:
It's moved to https://github.com/trinodb/trino/pull/10475/files/818dc2e0c4e98beb62ad5ab0ca2ceda32cd9e9d7#diff-c8f4753b54968c1b606f9044250705cbfeace0f4aba1d67740a3703919160f3cR177